public/js/app.js

```diff
@@ -951,6 +958,28 @@
     addChatEntry($outcome);
   };
 
+  // Send a private message to a user
+  var privateMessage = function(args, $outcome) {
+    if($.inArray(args[0], usersnicks) === -1) {
+      $outcome.text('(From binb): There\'s noone here by that name.');
+      return addChatEntry($outcome);
+    }
+    return addPrivate(args[0]);
+  };
+
+  // Reply to the last player who sent you a private message
+  var replyToPrivate = function(args, $outcome) {
+    if(!replyto) {
+      $outcome.text('(From binb): There\'s noone to reply to.');
+      return addChatEntry($outcome);
+    }
+    if($.inArray(replyto, usersnicks) === -1) {
+      $outcome.text('(From binb): They aren\'t here anymore.');
```

Comment:
Can we use a better message here? I see no reason for the plural since we are targeting a single player.